spec/models/payment_confirmation_spec.rb

```diff
@@ -43,6 +43,17 @@
 
       expect(payroll_run.reload.confirmation_report_uploaded_by).to eq(admin_user_id)
     end
+
+    it "queues emails" do
```

Comment:
Could this `it` be rewritten to be more specific about the behaviour we expect? e.g. it sends payment confirmation emails with a payment date of the following Friday?